daemon/daemon.go

```diff
@@ -13,10 +13,10 @@ import (
 
 	"github.com/weaveworks/flux"
 	"github.com/weaveworks/flux/api"
-	"github.com/weaveworks/flux/api/v10"
-	"github.com/weaveworks/flux/api/v11"
-	"github.com/weaveworks/flux/api/v6"
-	"github.com/weaveworks/flux/api/v9"
+	v10 "github.com/weaveworks/flux/api/v10"
```

Comment:
These packages should already have those names; i.e., they don't need aliases.

Reply:
Introduced by tooling as you assumed (VSC with no Golang related config except for `go.autocompleteUnimportedPackages` and `go.testFlags` set).

I will take a look and see if I can track down the thing that thinks this change should be made and shut it down.